src/main/webapp/dev/js/test/CommonTestFunctions.es6

```diff
@@ -33,20 +33,21 @@ function clearBootboxModalStub() {
     $('#test-bootbox-modal-stub').html('');
 }
 
+const bootboxStub = {
```

Comment:
I agree with @HirdayGupta haha; @LiHaoTan take a look at the rest of the file - there are other stubs too.